src/security/dependency-check/steps/npm-dependency-check.steps.yaml

```diff
@@ -0,0 +1,73 @@
+# Steps to run an NPM depdendency check
+parameters:
+  - name: path
+    default: ''
+  - name: projectName
+    default: 'Build NPM'
+  - name: publishAs
+    default: ''
+  - name: skip
```

Comment:
I don't like that we have a parameter that seemingly skips every task if true - can we remove it?

Reply:
Yep, removed this and updated the examples repo accordingly